### PR TITLE
add keras docs and fix GaussianSampler newly

### PR DESCRIPTION
**Pull request by yunyanliluo**

### CODE REVIEW

**Comment by yunyanliluo** [comment]
fix GaussianSampler from yesterday's bug

add keras docs:

./GlobalMaxPooling2D.scala
./SpatialDropout2D.scala
./GaussianSampler.scala
./Exp.scala
./Square.scala
./AtrousConvolution1D.scala
./BatchNormalization.scala
./UpSampling3D.scala

**Comment by yunyanliluo** [comment]
jenkins has passed: http://10.239.47.211:18888/job/ZOO-PR-Validation/1582/

**Comment by hkvision** [comment]
@hhbyyh Could you please have a review?

**Comment by hkvision** [comment]
@yangw1234 Please have a look at GaussianSampler. Especially, how many inputs to this layer and also is the Keras wrapping correct.

**Comment by hkvision** [comment]
@yunyanliluo Then could you please test the scala code, modify and confirm this?
We also need to specify the input size in inputShape

**Comment by hkvision** [comment]
Also @yangw1234 is it ok to use KerasLayerWrapper for this layer? Seems it won't check the input size, and if the input shape is not correct, it may throw some error that is not obvious.

**Comment by yangw1234** [comment]
@hkvision not really sure about the Wrapper. GaussianSampler take two tensors of the same size and output a tensor of that size. It seems quite straightforward to do the check.

**Comment by hkvision** [comment]
@yangw1234 Then we can just make it extend KerasLayer and override computeOutputShape instead of using KerasLayerWrapper? Maybe we can do this in a later stage if we have labor.

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 1]
```diff
@@ -1397,3 +1397,367 @@ array([[[ 0.06560206,  0.38629526, -0.23159817],
...
+GaussianSampler(inputShape = null)
+```
+**Python:**
+```python
+GaussianSampler(input_shape=None, name=None)
+```
+
+**Parameters:**
+
+* `inputShape`: Only need to specify this argument when you use this layer as the first layer of a model. For Scala API, it should be a [`MultiShape`](../keras-api-scala/#multishape) object that consists of two identical SingleShape. For Python API, it should be a list of two identical shape tuple. Batch dimension should be excluded.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: is there #multishape section in the page?

**Comment by hkvision** [comment]
Since there is code change, you need to run a jenkins job.

File: docs/docs/KerasStyleAPIGuide/Layers/core.md (hunk 1 above)
Comment (hkvision) [reply docs/docs/KerasStyleAPIGuide/Layers/core.md]: change back to #shape

**Comment by yunyanliluo** [comment]
I run a jenkins job successfully after code change, the link is http://10.239.47.211:18888/job/ZOO-PR-Validation/1637/
